test/e2e/tests/custom-rpc-history.spec.js

```diff
@@ -23,8 +23,9 @@ describe('Stores custom RPC history', function () {
         await driver.fill('#password', 'correct horse battery staple');
         await driver.press('#password', driver.Key.ENTER);
 
-        const rpcUrl = 'http://127.0.0.1:8545/1';
-        const chainId = '0x539'; // Ganache default, decimal 1337
+        const rpcUrl = 'https://rpc-mumbai.matic.today';
```

Comment:
Why was this url change needed? This would make our e2e tests dependent on network conditions, which we try to avoid. There might be a way to mock the behaviour this change is aiming to take advantage of.

Reply:
Yeah I figured that could be an issue. The reason I changed it was because in this test flow the ganache network is already a configured network and with the changes to validation in this PR it won't allow you to add the same network twice so this test wasn't able to validate the adding of network. I suppose we could delete and re-add the ganache network instead.

Reply:
@PeterYinusa and @danjm the prepopulated ganache network here doesn't appear to be tied to the state of the network controller in the fixtures json... is there a way to not have the ganache server present in the configured RPCs in the test env?

Reply:
I suppose a larger question that this raises is should this validation be (as the title of the PR currently suggests) just a warning and not an error that disables the ability to save duplicate networks (as the PR's actually currently functioning)?

Reply:
@danjm ^

Reply:
> I suppose a larger question that this raises is should this validation be (as the title of the PR currently suggests) just a warning and not an error that disables the ability to save duplicate networks (as the PR's actually currently functioning)?

I don't know about this. We should ask for opinions on this. Although, I think the way this PR is currently implemented is probably fine.

Reply:
Regarding how test having a different network.

ganache-cli allows this flag:

`--chainId: Specify the Chain ID ganache-cli will use for eth_chainId RPC and the CHAINID opcode. For legacy reasons, the default is currently 1337 for eth_chainId RPC and 1 for the CHAINID opcode. Setting this flag will align the chainId values. This will be fixed in the next major version of ganache-cli and ganache-core!`

We could maybe add the option to our test helpers to run two ganache servers at the same time, with different chain ids.

Reply:
There might be a simpler option: update the text fixture to remove ganache. It's no longer a "special" baked-in network - it's just present by default, and can be removed (IIRC).

Reply:
We later realized that my suggestion here doesn't make sense, because we want the initial network to be a local one to avoid the test depending on the network.